docker/omnibus-centos6/Dockerfile

```diff
@@ -58,4 +57,18 @@ RUN chown jenkins /var/cache/omnibus
 RUN chown jenkins /opt/metasploit-framework
 RUN chown -R jenkins /opt/chef/embedded/lib/ruby/gems
 
+RUN echo "#!/usr/bin/expect -f" > /usr/bin/signrpm && \
```

Comment:
You can probably use a here doc with command substitution here, but I don't want to muck with it and screw it up. Same for below.